chia/cmds/farm.py

```diff
@@ -105,3 +105,26 @@ def challenges_cmd(ctx: click.Context, farmer_rpc_port: Optional[int], limit: in
     from chia.cmds.farm_funcs import challenges
 
     asyncio.run(challenges(ChiaCliContext.set_default(ctx).root_path, farmer_rpc_port, limit))
+
+
+@farm_cmd.command("connect-solver", help="Connect to a solver")
+@click.option(
+    "-fp",
+    "--farmer-rpc-port",
+    help="Set the port where the Farmer is hosting the RPC interface",
```

Comment:
is the solver really interacting with the farmer RPC?
I would have expected the *farmer* to connect to the solver, not the other way around. And that they would talk the solver protocol, not RPC

Reply:
this is the port where your client calls the farmer to tell it to connect to the specific solver